### PR TITLE
Use ATen equivalents for variable element_size and nelement.

### PR DESCRIPTION
**Pull request by gchanan**

### CODE REVIEW

File: tools/autograd/templates/python_variable_methods.cpp [hunk 1]
```diff
@@ -72,6 +82,8 @@ PyMethodDef variable_methods[] = {
   {"contiguous", (PyCFunction)THPVariable_contiguous, METH_NOARGS, NULL},
   {"detach", (PyCFunction)THPVariable_detach, METH_NOARGS, NULL},
   {"detach_", (PyCFunction)THPVariable_detach_, METH_NOARGS, NULL},
+  {"nelement", (PyCFunction)THPVariable_numel, METH_NOARGS, NULL},
```
Comment (apaszke) [inline tools/autograd/templates/python_variable_methods.cpp]: this is also aliased as `numel`

Comment (gchanan) [reply tools/autograd/templates/python_variable_methods.cpp]: Right, ATen already has that so it works automagically.

**Review by colesbury — APPROVED** [review]
lgtm

File: tools/autograd/templates/python_variable_methods.cpp (hunk 1 above)
Comment (apaszke) [reply tools/autograd/templates/python_variable_methods.cpp]: Right, I didn't notice that it's autogenerated in `py_method_defs`

**Review by apaszke — APPROVED** [review]

**Pull request merged**